source/common/upstream/outlier_detection_impl.cc

```diff
@@ -249,9 +249,18 @@ DetectorConfig::DetectorConfig(const envoy::config::cluster::v3::OutlierDetectio
                                           DEFAULT_ENFORCING_CONSECUTIVE_LOCAL_ORIGIN_FAILURE))),
       enforcing_local_origin_success_rate_(static_cast<uint64_t>(
           PROTOBUF_GET_WRAPPED_OR_DEFAULT(config, enforcing_local_origin_success_rate,
-                                          DEFAULT_ENFORCING_LOCAL_ORIGIN_SUCCESS_RATE))),
-      max_ejection_time_ms_(static_cast<uint64_t>(
-          PROTOBUF_GET_MS_OR_DEFAULT(config, max_ejection_time, DEFAULT_MAX_EJECTION_TIME_MS))) {}
```

Comment:
Alternate implementation:
```
PROTOBUF_GET_MS_OR_DEFAULT(
    config, max_ejection_time,
    std::max(DEFAULT_MAX_EJECTION_TIME_MS, base_ejection_time_ms_))
```

Reply:
That is good idea. Implemented like that but the change required conversion of `DEFAULT_MAX_EJECTION_TIME_MS` from _const_ to _constexpr_.